<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1524</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnIncANN">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>195</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar e Treinar</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLins">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>2048</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCols">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>10</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>2048</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>15</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamanho</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N. Camadas</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCam">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>46</y>
      <width>52</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>211</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Treinamento</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>N. Amostras</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxAms">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>N. Testes(K)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTest">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>45</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidgetCam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>256</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEnd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>611</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/lele/Desenvolvimento/Programas/Teste/FotosLatasREXAM/Lata</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End. Imgs</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetDB">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>160</y>
      <width>331</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetClasses">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>260</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCarrDB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar DB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Processar</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCanais">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>52</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInic">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicializar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>71</width>
      <height>69</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonMLP">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>MLP</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonKNN">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>KNN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSVM">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>SVM</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonTeste">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>120</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Teste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_XIMEA">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>XIMEA</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgXIMEA">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Parar">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1524</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
